src/Symfony/Component/Notifier/Bridge/Zendesk/composer.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "symfony/zendesk-notifier",
+    "type": "symfony-notifier-bridge",
+    "description": "Symfony Zendesk Notifier Bridge",
+    "keywords": ["chat", "zendesk", "notifier"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Joseph Bielawski",
+            "email": "stloyd@gmail.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=8.1",
+        "symfony/http-client": "^5.4|^6.0",
+        "symfony/notifier": "^5.4|^6.0"
```

Comment:
The `UnsupportedSchemeException` in 5.4 does not know anything about this bridge, so it should need symfony/notifier 6.2